docs/apis-tools/console-api/tutorial.md

```diff
@@ -0,0 +1,204 @@
+
+```
+const accessToken = await getAccessToken();
+```
+
+3. As noted in the detailed API description in [Swagger](https://console.cloud.camunda.io/customer-api/openapi/docs/#/), you must call your Administration API URL and cluster ID. Ensure these credentials are added to your `.env` file, and script the references to the values. For example:
+
+```
+const administrationApiUrl = process.env.ADMINISTRATION_API_URL;
+  const clusterId = process.env.CLUSTER_ID;
+```
+
+4. Script the API endpoint to list the clients within your cluster:
+
+```
+const url = `${administrationApiUrl}/clusters/${clusterId}/clients`;
+```
+
+5. Document your GET request:
+
+```
+var options = {
+    method: "GET",
+    url,
+    headers: {
+      Accept: "application/json",
+      Authorization: `Bearer ${accessToken}`
+    }
+  };
+```
+
+6. Call the clients endpoint, process the results from the API call, emit the clients to output, and emit an error message from the server if necessary:
+
+```
+try {
+    // Call the clients endpoint.
+    const response = await axios(options);
+
+    // Process the results from the API call.
+    const results = response.data;
+
+    // Emit clients to output.
+    results.forEach(x => console.log(`Name: ${x.name}; ID: ${x.clientId}`));
+  } catch (error) {
+    // Emit an error from the server.
+    console.error(error.message);
+  }
+```
+
+7. In your terminal, run `npm run cli admin list` for a list of your existing clients.
+
+:::note
+This `list` argument is outlined at the bottom of the `administration.js` file, and executed with the `cli.js` file. While we will view, create, and delete clients in this tutorial, you may add additional arguments depending on the API calls you would like to make.
+:::
+
+If you have any existing clients, the `Name: {name}; ID: {Id}` will now output. If you have an invalid API name or action name, or no arguments provided, an error message will output as outlined in the `cli.js` file.
+
+## POST a client
+
+To create a new client, you will follow similar steps as outlined in your [GET request] (#get-clientid) above:
+
+1. Your function should now look like the following:
+
+```
+async function addClient([clientName])
+```
+
+2. Adjust your API endpoint to add a new client to a cluster:
+
+```
+const url = `${administrationApiUrl}/clusters/${clusterId}/clients`;
+```
+
+3. When configuring your API call, issue a POST request, and add a body containing information for the new client:
+
+```
+data: {
+      clientName: clientName
+    }
+```
+
+4. Process the results from the API call:
+
+```
+const newClient = response.data;
+```
+
+5. Emit the new client to output. While different from this example, you will likely want to capture the `clientSecret` property from the response, as this cannot be displayed again:
+
+```
+console.log(
+      `Client added! Name: ${newClient.name}. ID: ${newClient.clientId}.`
+    );
+```
```

Comment:
blocking: I think we should include all of the code in these steps, rather than just the difference between this function and the previous. I think there are too many opportunities for them to get something wrong if we don't.